versions/3.0.4.md

```diff
@@ -1557,9 +1528,35 @@ requestBody:
             properties: {}
 ```
 
-In this example, the contents in the `requestBody` MUST be stringified per [RFC1866](https://tools.ietf.org/html/rfc1866) when passed to the server.  In addition, the `address` field complex object will be stringified.
+In this example, the contents in the `requestBody` MUST be encoded per [RFC1866](https://tools.ietf.org/html/rfc1866) when passed to the server.  In addition, the `address` field complex object will be serialized to a string representation prior to encoding.
+
+When passing complex objects in the `application/x-www-form-urlencoded` content type, the default serialization strategy of such properties is described in the [Encoding Object](#encodingObject)'s [`style`](#encodingStyle) property as `form`.
+
+With this example, given an `id` of `f81d4fae-7dec-11d0-a765-00a0c91e6bf6` and a US-style address as follows:
+
+```json
+{
+  "streetAddress": "123 Example Dr.",
+  "city": "Somewhere",
+  "state": "CA",
+  "zip": 99999
```

Comment:
This is extremely minor/beside the point, but a zip should be a string as they can start with 0 (though, maybe not beside the point if wanting to show encoding of a number)

Reply:
Let's go with string- technically ZIP can also be something like 99999+1234, so I'll use that so it's not ambiguous.  Good catch!  It's always better if examples don't confuse with extraneous details.